apps/client/src/components/Location/ChangeAddressModal.tsx

```diff
@@ -1,16 +1,28 @@
+// @TODO: Rename this file to EditLocation
+// @TODO: Write test for this file
 import { Paragraph } from "@datapunt/asc-ui";
-import React from "react";
+import React, { useContext } from "react";
 
-import { ComponentWrapper } from "../../atoms";
-import { actions } from "../../config/matomo";
+import { ComponentWrapper, EditButton } from "../../atoms";
+import { actions, eventNames, sections } from "../../config/matomo";
+import { SessionContext, SessionDataType } from "../../context";
 import withTracking from "../../hoc/withTracking";
 import Modal from "../Modal";
 
 const ChangeAddressModal: React.FC<{
   matomoTrackEvent: Function;
-}> = ({ matomoTrackEvent }) => {
+  setActiveState: Function;
+  slug: string;
+  sttrFile: boolean;
```

Comment:
It's confusing to have the sttrFile having type boolean here. Either make it a string and do !! or use hasSTTR.

Reply:
Thanks! Replaced this